cookbooks/global_regional_coupling/doc/global_regional_coupling.md

```diff
@@ -0,0 +1,79 @@
+(sec:cookbooks:global_regional_coupling)=
+# Prescribing Velocities from Global Convection Models as Boundary Conditions in Regional Models
+
+*This section was contributed by Daniel Douglas.*
+
+In this cookbook we showcase how to use the python script `extract_local_velocity.py` (located in the [contrib/python/scripts](https://www.github.com/geodynamics/aspect/blob/main/contrib/python/scripts) directory) to prescribe velocity boundary conditions on a regional spherical chunk using the velocity solution from a global convection model. For the `extract_local_velocity.py` script to work, `pvpython` must be installed locally on your machine. The `pvpython` package is bundled with Paraview, and if you have Paraview on your machine the `pvpython` executable is located within the `bin` directory of your Paraview installation. The broad concept motivating this cookbook is the influence of far-field boundary conditions on processes occurring at regional scales, such as at a specific plate tectonics boundary. While regional geodynamic simulations commonly expand the size of a model domain to reduce the effects of free-slip or no-slip boundaries on the key system dynamics, this approach still does not take into account the effects far-field on regional dynamics. This cookbook demonstrates how to address this issue by extracting velocity constraints from a global mantle flow simulations and imposing the the extracted velocities along the boundaries of a regional 3D chunk simulation. In detail, we compare two regional instantaneous models in a chunk geometry, one applying velocity boundary conditions from a global convection model, the other applying free slip boundary conditions. Both cases use the S20RTS model as the initial temperature distribution, and the simple material model. The spherical chunk domain spans from -55$^&deg$ < latitude < -20$&deg$, 152$&deg$ < longitude < 210$&deg$, and 4770 km < radius < 6370 km.
+
+
+To start, run the global `initial-condition-S20RTS` cookbook for this demonstration, which has a relatively low resolution and run time. Assuming ASPECT has already been compiled, run this cookbook with the following terminal commands:
+
+```
+cd $ASPECT_SOURCE_DIR/cookbooks/initial-condition-S20RTS/
+../../build/aspect-release ./S20RTS.prm
+```
+
+Where `$ASPECT_SOURCE_DIR` is the path to your local ASPECT directory, and this assumes that the `build` directory is located in your `ASPECT_SOURCE_DIR`. This will generate a `solution.pvd` file required for the next step. Next, run the `extract_local_velocity.py` script after navigating to the correct directory using the following commands:
```

Comment:
```suggestion
Where `$ASPECT_SOURCE_DIR` is the path to your local ASPECT directory, and the `build` directory is assumed to be located in `ASPECT_SOURCE_DIR`. This will generate a `solution.pvd` file required for the next step. Next, run the `extract_local_velocity.py` script after navigating to the correct directory using the following commands:
```